cli/commands/list/list.go

```diff
@@ -187,7 +199,19 @@ func discoveredToListed(components component.Components, opts *Options) (ListedC
 			}
 		}
 
-		relPath, err := filepath.Rel(opts.WorkingDir, c.Path())
+		// Ensure component path is absolute for reliable relativity calculation
+		componentPath := c.Path()
+		if !filepath.IsAbs(componentPath) {
+			var err error
+
+			componentPath, err = filepath.Abs(componentPath)
```

Comment:
I think this is wrong. We never want to care about the PWD of the process. We need the path to always be relative to the `--working-dir` flag.